src/NLog/LayoutRenderers/WindowsIdentityLayoutRenderer.cs

```diff
@@ -31,7 +31,7 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 // 
 
-#if !SILVERLIGHT && !NETSTANDARD || NETSTANDARD1_3PLUS
+#if !SILVERLIGHT && !NETSTANDARD
```

Comment:
Isn't this a quick fix? Disabling the code?

Reply:
Sure is. But since it has been granted life by me, then I can also revoke its existence :)

Reply:
You can add a dependency on 4.4 in #2263 since it requires .NET Standard 2.0

Reply:
Netstandard 2.0 has a default package that includes almost everything but WCF. Only Netstandard 1.X needs explicit packages specified